lib/smart_app_launch/smart_access_brands_suite.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'smart_access_brands_group'
+
+module SMARTAppLaunch
+  class SMARTAccessBrandsSuite < Inferno::TestSuite
+    id 'smart_access_brands'
+    title 'SMART Access Brands STU2.2'
+    version VERSION
+
+    description <<~DESCRIPTION
+      The SMART User-access Brands Test Suite verifies that Brand Bundle Publishers publish valid User-access
+      Brand Bundles according to the SMART App Launch IG
+      [User-access Brands and Endpoints](https://hl7.org/fhir/smart-app-launch/brands.html#user-access-brands-and-endpoints)
+      requirements.
+
+      The specification defines FHIR profiles for Endpoint, Organization, and Bundle resources that help users connect
+      their apps to health data providers. It outlines the process for data providers to publish FHIR Endpoint and
+      Organization resources, where each Organization includes essential branding information such as the organization's
+      name, logo, and user access details. Apps present branded Organizations to help users select the right data
+      providers.
+
+      This Test Suite provides conformance testing to ensure the published User-access Brands Bundles are publicly
+      available and contain valid resources.
```

Comment:
I think we want to also note like this:

>  This test suite is currently designed to fetch and validate a single User-Access Brand Bundle.  It does not currently evaluate the system's ability to allow Health Data Providers to manage all data elements marked “Must-Support” in the “[User Access Brand](http://hl7.org/fhir/smart-app-launch/STU2.2/StructureDefinition-user-access-brand.html)” and “[User Access Endpoint](http://hl7.org/fhir/smart-app-launch/STU2.2/StructureDefinition-user-access-endpoint.html)” profiles.

I bring this up because we aren't doing "must support" checking here, right?  Purely "is this bundle valid".  Which is fine but we should not that, as it is a SHALL we do not test here.